### DISCUSSION TITLE
next release ?

### DISCUSSION THREAD

**Discussion by jhaesselbacher** [semaphoreui/semaphore#985] [Q&A]
Do you know when you plan to release the next version?
And what will be the next features?
Otherwise it's a great product, it does the job :)

**Comment by fiftin** [comment]
Hi,
What features to you need?

**Reply by cf-sewe** [reply]
An Ansible update would be highly appreciated. While 2.10 still works, it is marked as being deprecated by some modules and will lead to issues sooner or later. "Frequent" updates also imply that a project is still supported and can be relied on. 

see #1023
